crm_claim_rma/tests/test_picking_creation.py

```diff
@@ -142,3 +142,7 @@ def test_02_new_product_return(self):
         self.assertEquals(picking.location_dest_id,
                           self.warehouse_id.lot_stock_id,
                           "Incorrect destination location")
+
+    def test_copy(self):
+        new_claim = self.claim_id.copy()
+        self.assertNotEqual(new_claim.code, self.claim_id.code)
```

Comment:
yep, this is just to check that the code I removed was not required (and actually it was harmful since we were consuming twice from the crm code sequence).